src/Entity/Form/AppCreateForm.php

```diff
@@ -110,6 +110,10 @@ protected function alterFormWithApiProductElement(array &$form, FormStateInterfa
    * @param \Drupal\Core\Form\FormStateInterface $form_state
    *   The form state object.
    *
+   * @return array $elements
```

Comment:
Weird that PHPCS did not complain about this in tests, it should have. But if you add the `@return` tag, always describe for humans what a function/method returns. Here it contains the "The API product render element."

Reply:
Added desription to the @return tag in function comment